lib/process-services-cloud/src/lib/task/start-task/components/start-task-cloud.component.html

```diff
@@ -73,8 +73,16 @@
                     [title]="'ADF_CLOUD_TASK_LIST.START_TASK.FORM.LABEL.CANDIDATE_GROUP'"
                     [appName]="appName"
                     (selectGroup)="onCandiateGroupSelect($event)"
-                    (removeGroup)="onCandiateGroupRemove($event)"></adf-cloud-group>
-                <div fxFlex></div>
+                    (removeGroup)="onCandiateGroupRemove($event)">
+                </adf-cloud-group>
+
+                <mat-form-field fxFlex>
```

Comment:
Maybe is better put this list of forms in a subcomponent can be useful somewhere else

Reply:
This feature will be covered as part of https://issues.alfresco.com/jira/browse/ADF-4349

Reply:
can we start to decompose it now? More we are able to create the right separation easy will be reusing it tomorrow

Reply:
Let's follow the agile way. Do not anticipate. We already have another issue and in case it necessary we will create a  component

Reply:
@arditdomi can take it